README.md

```diff
@@ -1,34 +1,42 @@
-
-
 # Hello and Welcome
 ## It should be a fun time doing the codez with all yall!
 ### You can check out tutorials such as <a href='https://github.com/firstcontributions/first-contributions/blob/master/README.md'>this one</a> to get famaliar with the git hubz
 
-
 # Easy to git
-The idea of this repo is to get you started with git and GitHub.
-
+The idea of this repo is to get you started with Git and GitHub.
 ## The very basics
-git and GitHub are different tools but work together. git is a program you run on your machine. And you use git to make and get changes to code stored on GitHub. GitHub is a remote host of files, that you work on with other people.
-
-### So can you repeat, how are those different?
-Because GitHub is on a different computer, when we create our remote repository, we set permissions for people to see the code, make copies or "fork" it, and "push" our changes to it.
-So the remote repository automatically becomes a collaboration tool. While git controls the project, we need a remote repository to collaborate, and that is GitHub.
-
+Git and GitHub are different tools but work together. Git is a program you run on *your machine*, and it's useful to interact with the remote code-repository. This remote host of files is GitHub.
```

Comment:
You really get it. Thanks for putting so much thought into cutting down on bulk and making clear explanations.